src/main/java/com/training/web/pages/flipkart/ProductPage.java

```diff
@@ -0,0 +1,58 @@
+package com.training.web.pages.flipkart;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ProductPage {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    private By productName1=By.className("G6XhRU");
+    private By name = By.className("B_NuCI");
+    private By size=By.id("swatch-0-size");
+    private By cart = By.xpath("//button[contains(@class,'_2KpZ6l')]");
+    private By price= By.xpath("//div[contains(@class,'_30jeq3')]");
+    private By loaderIcon = By.xpath("//div[@class='_2YsvKq _3bgaUQ']/*[name()='svg']");
+
+    public ProductPage(WebDriver driver) {
+        this.wait = new WebDriverWait(driver, 30);
+        this.driver = driver;
+    }
+
+    public ProductPage clickSize() {
```

Comment:
Lets make this method more reusable by providing the size to select - `clickSize(String size)`